.gitignore

```diff
@@ -24,7 +24,6 @@ perl/Makefile.config
 /doc/manual/src/command-ref/new-cli
 /doc/manual/src/command-ref/conf-file.md
 /doc/manual/src/command-ref/experimental-features-shortlist.md
-/doc/manual/src/contributing/experimental-feature-descriptions.md
```

Comment:
This got added in [bc192a9](https://github.com/NixOS/nix/commit/bc192a95ef4c777d9ca0ad025e0fb2b7f18879f7), although I'm not certain how we'd update the description of experimental features then?

Reply:
Oh, never mind… it is generated. 😁